pom.xml

```diff
@@ -208,6 +210,14 @@
           <suppressionsLocation>${basedir}/src/main/resources/checkstyle/checkstyle-suppressions.xml</suppressionsLocation>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>com.github.spotbugs</groupId>
+        <artifactId>spotbugs-maven-plugin</artifactId>
+        <version>${spotbugs.plugin.version}</version>

```

Comment:
Here you can use `${commons.spotbugs.version}` as it is controlled in commons-parent

Reply:
done